runners/gradle-plugin/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/kotlinClasspathUtils.kt

```diff
@@ -39,7 +39,7 @@ private fun Project.compileClasspathOf(compilation: KotlinCompilation): FileColl
     if (compilation.target.isAndroidTarget()) {
         // This is a workaround for https://youtrack.jetbrains.com/issue/KT-33893
         @Suppress("DEPRECATION") // for compatibility
-        return compilation.compileKotlinTask.cast<KotlinCompile>().classpath
+        return compilation.compileKotlinTask.cast<KotlinCompile>().libraries
```

Comment:
`classpath` is deprecated with error, so no other choice.

`libraries` exists only in KGP >= 1.7.0, so Dokka 1.8 is not backwards compatible with Kotlin (KGP) 1.6.21 and earlier.

This needs to be discussed additionally and stressed in documentation or somewhere. I've asked the build tools team for more info until then